dpnp/dpnp_iface_manipulation.py

```diff
@@ -309,6 +310,57 @@ def atleast_3d(*arys):
         return res
 
 
+def broadcast_arrays(*args, subok=False):
+    """
+    Broadcast any number of arrays against each other.
+
+    For full documentation refer to :obj:`numpy.broadcast_arrays`.
+
+    Returns
+    -------
+    broadcasted : list of dpnp.ndarray
+        These arrays are views on the original arrays.
+
+    Limitations
+    -----------
+    Parameter `args` is supported as either :class:`dpnp.ndarray`
+    or :class:`dpctl.tensor.usm_ndarray`.
+    Otherwise ``TypeError`` exception will be raised.
+    Parameter `subok` is supported with default value.
+    Otherwise ``NotImplementedError`` exception will be raised.
+
+    See Also
+    --------
+    :obj:`dpnp.broadcast_to` : Broadcast an array to a new shape.
+
+    Examples
+    --------
+    >>> import dpnp as np
+    >>> x = np.array([[1, 2, 3]])
+    >>> y = np.array([[4], [5]])
+    >>> np.broadcast_arrays(x, y)
+    [array([[1, 2, 3],
+            [1, 2, 3]]), array([[4, 4, 4],
+            [5, 5, 5]])]
+
+    """
+    if subok is not False:
```

Comment:
leave an empty line between comment block and the code
```suggestion
    """

    if subok is not False:
```